<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>601</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 28pt &quot;News706 BT&quot;;</string>
     </property>
     <property name="text">
      <string>Ambulance Management System</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>571</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);
border-radius:16px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>440</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;News706 BT&quot;;
border-radius:5px;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:0.966, stop:0.0795455 rgba(9, 169, 0, 255), stop:0.522727 rgba(0, 255, 50, 255), stop:0.931818 rgba(15, 155, 0, 255));</string>
      </property>
      <property name="text">
       <string>Request</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;News706 BT&quot;;</string>
      </property>
      <property name="text">
       <string>Name: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;News706 BT&quot;;</string>
      </property>
      <property name="text">
       <string>CNIC: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>210</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;News706 BT&quot;;</string>
      </property>
      <property name="text">
       <string>Cell No.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #000000;
background-color: rgb(255, 255, 255);
font: 75 16pt &quot;News706 BT&quot;;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #000000;
background-color: rgb(255, 255, 255);
font: 75 16pt &quot;News706 BT&quot;;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #000000;
background-color: rgb(255, 255, 255);
font: 75 16pt &quot;News706 BT&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;News706 BT&quot;;</string>
      </property>
      <property name="text">
       <string>District: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;News706 BT&quot;;</string>
      </property>
      <property name="text">
       <string>Area: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>390</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;News706 BT&quot;;</string>
      </property>
      <property name="text">
       <string>Emergency: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>260</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 75 18pt &quot;News706 BT&quot;;
border-radius:8px;
border: 1px solid #000000;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>320</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 75 18pt &quot;News706 BT&quot;;
border-radius:8px;
border: 1px solid #000000;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>380</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 75 18pt &quot;News706 BT&quot;;
border-radius:8px;
border: 1px solid #000000;</string>
      </property>
      <item>
       <property name="text">
        <string>Medical</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Road Accident</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fire Burn</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Electric Shock</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blood Pressure</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Allergy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Terror Attack</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 24pt &quot;News706 BT&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Emergency Request Form</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="backBtn">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>520</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;News706 BT&quot;;
border-radius:5px;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:0.966, stop:0.0795455 rgba(9, 169, 0, 255), stop:0.522727 rgba(0, 255, 50, 255), stop:0.931818 rgba(15, 155, 0, 255));
background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:0.966, stop:0.0795455 rgba(0, 133, 169, 255), stop:0.522727 rgba(0, 255, 255, 255), stop:0.931818 rgba(0, 109, 155, 255));</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>520</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;News706 BT&quot;;
border-radius:5px;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:0.966, stop:0.0795455 rgba(169, 0, 0, 255), stop:0.522727 rgba(255, 0, 0, 255), stop:0.931818 rgba(155, 0, 0, 255));</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
